src/core/utils/share-style.ts

```diff
@@ -160,3 +160,17 @@ export const allowsHoverText = (allowsHover: boolean, isLight: boolean, active:
           : 'transparent',
       }
     : undefined;
+
+export const ButtonPickerStyle = (isLight: boolean, selected: boolean) => ({
+  background: isLight ? (selected ? '#1AAB9B' : 'transparent') : selected ? '#098C7D' : 'transparent',
+
+  borderColor: isLight ? (selected ? '#1AAB9B' : '#D4D9E1') : selected ? '#098C7D' : '#708390',
+  boxShadow: isLight
+    ? selected
+      ? '2px 4px 7px rgba(26, 171, 155, 0.25)'
+      : 'none'
+    : selected
+    ? '2px 4px 7px rgba(26, 171, 155, 0.25)'
+    : 'none',
```

Comment:
this could be more readable if you use the `selected` value as follow:

```
boxShadow: selected ? (isLight ? '2px 4px 7px rgba(26, 171, 155, 0.25)' : '2px 4px 7px rgba(26, 171, 155, 0.25)') : 'none'
```

But I noted that the shadows used for light and dark modes are the same, so this should be as follow:
```
boxShadow: selected ? '2px 4px 7px rgba(26, 171, 155, 0.25)' : 'none'
```

Reply:
I agree